lib/learn_submit/submission.rb

```diff
@@ -45,145 +43,62 @@ def user
 
     private
 
-    def simulate_submission!(retries=3)
-      begin
-        repo_name = git.repo_name(remote: 'origin')
-        org_name  = git.org_name(remote: 'origin')
-
-        if retries == 3
-          puts 'Pushing changes to GitHub...'
-          sleep(1)
-          puts 'Submitting lesson...'
-        end
-
-        Timeout::timeout(15) do
-          client.submit_event(
-            event: 'pull_request',
-            action: 'opened',
-            learn_oauth_token: token,
-            repo_name: repo_name,
-            base_org_name: org_name,
-            forkee: { full_name: nil },
-            pull_request: {
-              head: {
-                repo: {
-                  full_name: "#{org_name}/#{repo_name}",
-                  name: repo_name
-                }
-              }
-            },
-            summary: {
-              action: 'opened'
-            }
-          )
-        end
-
-        puts 'Done.'
-        after_ide_submission(repo_name)
-      rescue Timeout::Error
-        if retries > 0
-          puts "There was a problem submitting this lab. Retrying..."
-          simulate_submission!(retries-1)
-        else
-          puts "There is an issue connecting to Learn. Please try again."
-          File.write(file_path, 'ERROR: Error connecting to Learn')
-          exit
-        end
-      end
-    end
-
     def commit_and_push!
       File.write(file_path, 'Pushing your code to GitHub...')
 
-      if dot_learn && dot_learn['github'] == false
-        git.commit
-      else
-        git.commit_and_push
-      end
+      git.commit_and_push
 
       # Just to give GitHub a second to register the repo changes
       sleep(1)
     end
 
     def submit!(retries=3)
-      if dot_learn && dot_learn['github'] == false
-        simulate_submission!
-      else
-        if retries >= 2
-          puts 'Submitting lesson...'
-          File.write(file_path, 'Opening a Pull Request...')
-        end
-        repo_name   = git.repo_name
-        branch_name = git.branch_name
-        sleep(1)
+      if retries >= 2
+        puts 'Submitting lesson...'
+        File.write(file_path, 'Opening a Pull Request...')
+      end
+      repo_name   = git.repo_name
+      branch_name = git.branch_name
+      sleep(1)
```

Comment:
Not sure why we need this `sleep(1)` (https://github.com/learn-co/learn-submit/commit/c16d3acb08c6360b4030ada032fb1c64522cabd9) but leaving it in for now